libs/watt/package/menu/watt-menu.component.ts

```diff
@@ -119,7 +116,20 @@ export class WattMenuComponent {
    */
   hasIcons = computed(() => this.menuItems().some((item) => item.hasIcon()));
```

Comment:
question: it looks like all this "has icons" logic only applies to styles. have you considered a CSS-only solution using the `:has` selector?